### DISCUSSION TITLE
open /home/michael/.scrypted/docker-compose.yml: permission denied

### DISCUSSION THREAD

**Discussion by mbierman** [koush/scrypted#866] [Q&A]
Running the script, I get the error above: 

<img width="736" alt="image" src="https://github.com/koush/scrypted/assets/1205471/36d89c7e-82a2-4ab0-a5f0-e1c2103ff01c">

**Comment by loxK** [comment]
Same issue over here, Ubuntu Jammy.

My user is in the docker group, I can `docker ps`.

**Comment by bjia56** [comment]
Try `sudo chown -R $USER:$USER ~/.scrypted`

**Reply by loxK** [reply]
It is already owned by my user.

**Comment by meven3000** [comment]
Seems like some permission incompatibility with docker compose, even after adding the user to docker group, full perms on the folder it was still failing. 

reverted to https://github.com/koush/scrypted/wiki/Installation:-Docker-Linux

**Comment by meven3000** [comment]
After some more troubleshooting:

Assuming you are using Ubuntu! and installed using Docker using ubuntu methods/during install, then it uses SNAP.
I've had numerous issues with SNAP variants before and done some digging, hopefully this should work.


sudo snap remove docker
## If cannot remove then use -> sudo snap remove docker --purge

**sudo snap remove --purge docker

sudo rm -rf /var/lib/docker /etc/docker
sudo groupdel docker
sudo rm -rf /var/run/docker.sock

##Install the normal docker package
curl -fsSL https://get.docker.com -o get-docker.sh
sudo sh ./get-docker.sh --dry-run

#Assuming that there were no errors in the dry run to fix
sudo sh ./get-docker.sh

##wait for install to complete

##Add the required/running users who's going to be running the package
(Depending on you OS/User security requirements this may change))

sudo groupadd docker
sudo usermod -aG docker $USER
newgrp - docker

##Check that the current user can run docker containers
docker run hello-world

##Install Scrypted using docker compose

mkdir -p ~/.scrypted
curl -s https://raw.githubusercontent.com/koush/scrypted/main/install/docker/install-scrypted-docker-compose.sh > ~/.scrypted/install-scrypted-docker-compose.sh 
sudo SERVICE_USER=$USER bash ~/.scrypted/install-scrypted-docker-compose.sh
